framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -1051,6 +1051,12 @@ protected boolean visitWildcardSubtype(
         final Types types = subtype.atypeFactory.getProcessingEnv().getTypeUtils();
         final Elements elements = subtype.atypeFactory.getProcessingEnv().getElementUtils();
 
+        if (subtype.getKind() == TypeKind.NULL) {
```

Comment:
Is this actually the correct thing to do or just a hack to work around the crash?
Can you add some indication either way?